opendebates/context_processors.py

```diff
@@ -63,6 +63,7 @@ def _get_categories():
         'ALLOW_SORTING_BY_VOTES': mode.allow_sorting_by_votes,
         'ALLOW_VOTING_AND_SUBMITTING_QUESTIONS': mode.allow_voting_and_submitting_questions,
         'DEBATE_TIME': mode.debate_time,
+        'PREVIOUS_DEBATE_TIME': mode.previous_debate_time,
```

Comment:
Since per Ethan's comment this rightly should involve a bit of logic, I kept this line but updated in the way suggested.